perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -241,11 +255,68 @@ def SetFiles(self):
       self.RemoteCommand('echo "%s" | sudo tee %s' %
                          (value, path))
 
+  def ApplySysctlPersistent(self, key, value):
+    """Apply "key=value" pair to /etc/sysctl.conf and reboot.
+
+    The reboot ensures the values take effect and remain persistent across
+    future reboots.
+
+    Args:
+      key: a string - the key to write as part of the pair
+      value: a string - the value to write as part of the pair
+    """
+
+    self.RemoteCommand('sudo bash -c \'echo "%s=%s" >> /etc/sysctl.conf\''
+                       % (key, value))
+
+    self._needs_reboot = True
+
   def DoSysctls(self):
-    """Apply --sysctl to the VM."""
+    """Apply --sysctl to the VM.
+
+       The Sysctl pairs are written persistently so that if a reboot
+       occurs, the flags are not lost.
+    """
+
+    for pair in FLAGS.sysctl:
+      key, value = pair.split('=')
+      self.ApplySysctlPersistent(key, value)
+
+  def DoConfigureNetworkForBBR(self):
+    """Apply --network_enable_BBR to the VM."""
+
```

Comment:
die

Reply:
I don't follow 100% ... You mean no blank after doc string?